services/spark-operator/1.1.6/defaults/cm.yaml

```
@@ -5,4 +5,17 @@ metadata:
   name: spark-operator-1.1.6-d2iq-defaults
   namespace: ${releaseNamespace}
 data:
-  values.yaml: ""
+  values.yaml: |-
+    historyServer:
+      enabled: true
```

```suggestion
      enabled: false
```
Given the conversation [above](https://github.com/mesosphere/dkp-catalog-applications/pull/6/files#r759604314) about defaults.